components/prism-dot.js

```diff
@@ -0,0 +1,77 @@
+// https://www.graphviz.org/doc/info/lang.html
+
+(function (Prism) {
+
+	var ID = '(?:' + [
+		// an identifier
+		/\b[a-zA-Z_][\w\x80-\x9F\xA1-\xFF]*/.source,
```

Comment:
An ID [can](https://graphviz.org/doc/info/lang.html) begin with a `[A-Z_a-z\x80-\xFF]` character. In case of using `[\x80-\xFF]` as a first character, the `\b` pattern can’t handle a word boundary.

Reply:
I did this to guarantee O(n) runtime but you're right, it needs a better solution.